src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -508,6 +506,10 @@ class ADIOSWriter : public ILightweightPlugin
             mpi_pos = gc.getPosition();
             mpi_size = gc.getGpuNodes();
 
+            /* if number of aggregators is not set we use all mpi process as aggregator*/
+            if( mThreadParams.adiosAggregators == 0 )
+               mThreadParams.adiosAggregators=mpi_size.productOfComponents();
```

Comment:
please change the value to what I recommended since

> Aggregators are intresting for 10k+ cores, I am not sure if we can leave aggregators unset on small systems (<5k write processes). 

is not true :)